test/format/format.sh

```diff
@@ -30,6 +30,7 @@ usage() {
 	echo "    -j parallel  jobs to execute in parallel (defaults to 8)"
 	echo "    -n total     total jobs to execute (defaults to no limit)"
 	echo "    -R           run timing stress split test configurations (defaults to off)"
+	echo "    -r binary    record with UndoDB (defaults to no recording)"
```

Comment:
Maybe "record with UndoDB binary" to make it extra clear what the argument to `-r` is supposed to be.

Reply:
71b0c5a